build.gradle

```diff
@@ -1,15 +1,15 @@
 buildscript {
     ext {
         // Libraries
-        kotlinVersion = '1.3.21'
+        kotlinVersion = '1.3.50'
     }
 
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath 'com.android.tools.build:gradle:3.5.0'
```

Comment:
Please always check weather the gralde version you pick is compatible with fdroid.

Reply:
Where can we see the versions that are compatible with fdroid?

Reply:
I've no clue how to do it

Reply:
You can find a list of supported gradle versions [here](https://gitlab.com/fdroid/fdroidserver/blob/master/makebuildserver#L149). Apparently 3.5.0 is supported :)